guides/common/modules/proc_configuring-freeipa-authentication-on-server.adoc

```diff
@@ -104,6 +41,7 @@ The attack is possible even if the user did not previously enter the {Project} l
 # {foreman-maintain} service restart
```

Comment:
I don't see a good reason to restart services here. While you're reviewing, I think this can be dropped. And if it's still needed, we need to figure out why and fix that bug.

Reply:
You're right, it works even without the restart. Addressed.